server/jetstream_cluster.go

```diff
@@ -155,7 +155,6 @@ type consumerAssignment struct {
 type streamPurge struct {
 	Client  *ClientInfo              `json:"client,omitempty"`
 	Stream  string                   `json:"stream"`
-	LastSeq uint64                   `json:"last_seq"`
```

Comment:
Was only used in the below `isRecovering + Compact` block, so have cleaned it up.